src/main/java/org/rumbledb/runtime/functions/base/Functions.java

```diff
@@ -123,19 +123,7 @@
 import org.rumbledb.runtime.functions.sequences.value.DeepEqualFunctionIterator;
 import org.rumbledb.runtime.functions.sequences.value.DistinctValuesFunctionIterator;
 import org.rumbledb.runtime.functions.sequences.value.IndexOfFunctionIterator;
-import org.rumbledb.runtime.functions.strings.ConcatFunctionIterator;
-import org.rumbledb.runtime.functions.strings.ContainsFunctionIterator;
-import org.rumbledb.runtime.functions.strings.EndsWithFunctionIterator;
-import org.rumbledb.runtime.functions.strings.MatchesFunctionIterator;
-import org.rumbledb.runtime.functions.strings.NormalizeSpaceFunctionIterator;
-import org.rumbledb.runtime.functions.strings.StartsWithFunctionIterator;
-import org.rumbledb.runtime.functions.strings.StringFunctionIterator;
-import org.rumbledb.runtime.functions.strings.StringJoinFunctionIterator;
-import org.rumbledb.runtime.functions.strings.StringLengthFunctionIterator;
-import org.rumbledb.runtime.functions.strings.SubstringAfterFunctionIterator;
-import org.rumbledb.runtime.functions.strings.SubstringBeforeFunctionIterator;
-import org.rumbledb.runtime.functions.strings.SubstringFunctionIterator;
-import org.rumbledb.runtime.functions.strings.TokenizeFunctionIterator;
+import org.rumbledb.runtime.functions.strings.*;
```

Comment:
If I may, we agreed against using wildcard imports in the project some time ago. I have verified that this isn't picked up by spotless for the time being but I will investigate if we can add this to our configurations as well. For the time being, it would be great if we could keep explicit imports as they are.

If you are using IntelliJ IDEA, [this link](https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports) can be helpful. Otherwise, I'm sure other IDE's support something similar as well.

Reply:
Thanks, @CanBerker 

Could you also comment on the parser files? Were they also post-processed with IntelliJ to remove the wildcards (in this case, I do not think they were changed, so their change can simply be reverted)?

Reply:
Gladly. I was curious about these as well. Once formatted by spotless, as you mentioned, all changes except the wildcards disappear. It's extremely likely the wildcards used by ANTLR were removed by IDEA at some point.

Reply:
Thank you -- it would be ideal if we could find a way to do the same with eclipse, in order to stick to the no-wildcards convention. An alternative is to completely remove these automatically generated files from the repository, as previously suggested by Stefan. Then there is no need to worry about it any more (but it may require setting up the test queue appropriately).

Reply:
I have looked into this but it appears that Eclipse does not expose a configuration that we can use in a project-wide manner. Neither eclipse JDT through the spotless plugin, nor the eclipse IDE itself exposes such an option. 

We could consider removing the auto-generated and discuss its tradeoffs. Yet, for this case, I believe it would not help anything. The wildcard imports popped up in the Functions.java and they can appear basically in any file. Removing auto-generated files simply reduce the number of files wildcard imports could pop up in by 4. 

For the time being, I am planning to add guidelines for eclipse and IDEA configurations to the Rumble's wiki as I do not see a programmatic solution presently available. For the time being, I believe we have to stick to peer-reviews in enforcing this formatting decision.

Reply:
I noticed this change as well. It appears using code completion for classes not yet imported automatically changes the explicit imports for classes associated with that package to a wildcard import, at least given the default settings in IntelliJ. I'll pay attention to this for future pull requests.

Reply:
Thanks for your attention on the matter. Now that you have applied spotless, the only changes that remain in the auto-generated files seem to be the wildcard imports. If you have applied the configuration as shown in this [wiki page](https://github.com/RumbleDB/rumble/wiki/Wildcard-imports), running `IntelliJ - optimize imports` on auto-generated files should remove the wild-card imports. This should hopefully bring the files back to the original state. If all goes well, changes in the auto-generated files in this PR should disappear.